### PR TITLE
EOS-18020: Fix for alert event filters

### PR DESCRIPTION
**Pull request by shindeam**
Signed-off-by: Amol Shinde <amol.shinde@seagate.com>

## Problem Statement
<pre>
  <code>
  Story Ref (if any):
    EOS-18020: Alert event filter issues found during testing the integration of system
  </code>
</pre>
## Unit testing on RPM done
<pre>
  <code>
  Yes
  </code>
</pre>
## Problem Description
<pre>
  <code>
    Fix code issues which was failing in testing the alert generator filters.
  </code>
</pre>
## Solution
<pre>
  <code>
    Fixed code issues which was failing in testing the alert generator filters.
  </code>
</pre>
## Unit Test Cases
<pre>
  <code>
    [root@ssc-vm-c-0281 alert]# python3 test_node_alert_event_filter.py
     ********Alert Event Filter********
     Alert event filter test pass successfully
  </code>
</pre>

### CODE REVIEW

**Comment by ArchanaLimaye** [comment]
Please add UT details in PR section "Unit Test Cases"

File: ha/alert/filter.py [hunk 1]
```diff
@@ -57,9 +58,9 @@ def __init__(self):
         super().__init__()
 
         # Get filter type and resource types list from the alert monitor rule file
-        self.alert_filter_components_list = Conf.get(const.ALERTS.PK_ALERT_EVENT_COMPONENTS)
-        self.alert_filter_modules_list = Conf.get(const.ALERTS.PK_ALERT_EVENT_COMPONENT_MODULES)
-        self.alert_filter_module_operations = Conf.get(const.ALERTS.PK_ALERT_EVENT_OPERATIONS)
+        self.alert_filter_components = Conf.get(ALERT_EVENT_INDEX, const.ALERTS.PK_ALERT_EVENT_COMPONENTS)
```
Comment (ArchanaLimaye) [inline ha/alert/filter.py:61]: You can import the ALERTS module from const and use that directly

**Comment by indrajitzagade** [comment]
> 
> 
> Signed-off-by: Amol Shinde [amol.shinde@seagate.com](mailto:amol.shinde@seagate.com)
> ## Problem Statement
> 
>   ```
> 
>   Story Ref (if any):
>     Alert event filter issues found during testing the integration of system
>   
> ```
> 
> 
> ## Unit testing on RPM done
> 
>   ```
> 
>   Yes
>   
> ```
> 
> 
> ## Problem Description
> 
>   ```
> 
>     Fix code issues which was failing in testing the alert generator filters.
>   
> ```
> 
> 
> ## Solution
> 
>   ```
> 
>     Fixed code issues which was failing in testing the alert generator filters.
>   
> ```
> 
> 
> ## Unit Test Cases
> 
>   ```
> 
>     Unit Testing details here...
>   
> ```

@shindeam Please add unit testing details. Also, add ticket number in the PR title.

File: ha/alert/filter.py (hunk 1 above)
Comment (shindeam) [reply ha/alert/filter.py]: Done

**Comment by shindeam** [comment]
> > Signed-off-by: Amol Shinde [amol.shinde@seagate.com](mailto:amol.shinde@seagate.com)
> > ## Problem Statement
> > ```
> > 
> > Story Ref (if any):
> >   Alert event filter issues found during testing the integration of system
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > ## Unit testing on RPM done
> > ```
> > 
> > Yes
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > ## Problem Description
> > ```
> > 
> >   Fix code issues which was failing in testing the alert generator filters.
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > ## Solution
> > ```
> > 
> >   Fixed code issues which was failing in testing the alert generator filters.
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > ## Unit Test Cases
> > ```
> > 
> >   Unit Testing details here...
> > ```
> 
> @shindeam Please add unit testing details. Also, add ticket number in the PR title.

Done.

**Review by indrajitzagade — APPROVED** [review]

**Review by ajay-paratmandali — APPROVED** [review]

**Review by ArchanaLimaye — APPROVED** [review]

**Pull request merged**